src/integrationTest/sam.test.ts

```diff
@@ -120,16 +116,13 @@ async function closeAllEditors(): Promise<void> {
     await vscode.commands.executeCommand('workbench.action.closeAllEditors')
 }
 
-function validateLocalInvokeResult(
-    actualResult: LocalInvokeCodeLensCommandResult,
-    expectedResult: LocalInvokeCodeLensCommandResult
-) {
-    assert.strictEqual(actualResult.datum.name, expectedResult.datum.name)
-    assert.strictEqual(actualResult.datum.value, expectedResult.datum.value)
-    assert.strictEqual(actualResult.datum.unit, expectedResult.datum.unit)
+function validateLocalInvokeResult(actualResult: MetricDatum, expectedResult: MetricDatum) {
+    assert.strictEqual(actualResult.MetricName, expectedResult.MetricName)
+    assert.strictEqual(actualResult.Value, expectedResult.Value)
+    assert.strictEqual(actualResult.Unit, expectedResult.Unit)
 
-    expectedResult.datum.metadata!.forEach((value, key) => {
-        assert.strictEqual(actualResult.datum.metadata!.get(key), value)
+    expectedResult.Metadata!.forEach((key, entry) => {
```

Comment:
key and entry appear to be reversed from the array forEach signature
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
Weird, fixed